### PR TITLE
[Jotai] atomWithProxy sync option docs

### PR DESCRIPTION
**Pull request by tarngerine**
For: https://github.com/pmndrs/jotai/pull/752

### CODE REVIEW

File: docs/jotai/integrations/valtio.mdx [hunk 1]
```diff
@@ -48,6 +48,22 @@ const Counter: React.FC = () => {
...
+
+**proxyObject** (required): the Valtio proxy object you want to derive the atom from
+
+**options.sync** (optional): makes the atom update synchronously instead of waiting for batched updates, similar to [`valtio/useSnapshot`](https://github.com/pmndrs/valtio#update-synchronously). This will result in more renders, but have more guarantees that it syncs with other Jotai atoms.
+
+```
+atomWithProxy(proxyObject, { sync: true })
+```
+
+You may
```
Comment (dai-shi) [inline docs/jotai/integrations/valtio.mdx:65]: ??

**Comment by dai-shi** [comment]
Converting to draft as we will merge this after the release.

**Review by dai-shi — APPROVED** [review]
Thanks!!!

**Comment by tarngerine** [comment]
Where are the docs now?

**Comment by dai-shi** [comment]
In the same repo: https://github.com/pmndrs/jotai

**Comment by tarngerine** [comment]
done!

**Pull request closed without merging**